.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
These are hooks/lints that are not related to ansible-lint at all.

Reply:
True, can be done in another change.